source/transactions/tests/error-labels.yml

```diff
@@ -200,7 +200,7 @@ tests:
         result:
           # Note, the server will return the errorLabel in this case.
           errorLabelsContain: ["TransientTransactionError"]
-          errorLabelsOmit: ["UnknownTransactionCommitResult"]
+          errorLabelsOmit: ["RetryableWriteError", "UnknownTransactionCommitResult"]
```

Comment:
This test should already pass, just including the RetryableWriteError label to make it consistent with the rest of the tests in this file.